modules/combined/examples/phase_field-mechanics/hex_grain_growth_2D_eldrforce.i

```diff
@@ -161,7 +161,7 @@
   [./Periodic]
     [./All]
       auto_direction = 'x y'
-      variable = 'gr0 gr1 gr2 gr3 gr4 gr5 gr6 gr7 gr8 gr9 gr10 gr11 gr12 gr13 gr14'
+      variable = 'gr0 gr1 gr2 gr3 gr4 gr5 gr6 gr7'
```

Comment:
Hmm, this syntax should be cleaned up. We have a utility for this but I guess it needs to be added to the PeriodicBoundary objects. Nothing for you to do here, I'll open a new ticket.